<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewComponent</class>
 <widget class="QDialog" name="NewComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fill in component properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="cname"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Direction</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Point to:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QComboBox" name="pointTo">
       <item>
        <property name="text">
         <string>NA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Up</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Down</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Right</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Front</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Back</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>x:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="12" column="2" colspan="2">
      <widget class="QLineEdit" name="bendX">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>y:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="13" column="2" colspan="2">
      <widget class="QLineEdit" name="bendY">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>z:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="14" column="2" colspan="2">
      <widget class="QLineEdit" name="bendZ">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Axis</string>
       </property>
      </widget>
     </item>
     <item row="17" column="2">
      <widget class="QComboBox" name="axisConjunction"/>
     </item>
     <item row="7" column="2">
      <widget class="QLineEdit" name="sizeX">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLineEdit" name="sizeY">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLineEdit" name="sizeZ">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>x:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>y:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>z:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Unit</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="csizeUnit">
       <item>
        <property name="text">
         <string>cm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>meter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>km</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>inches</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>feet</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="stretch">
       <property name="text">
        <string>Stretch</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="16" column="2">
      <widget class="QComboBox" name="axisParent"/>
     </item>
     <item row="15" column="2">
      <widget class="QComboBox" name="myAxis"/>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Connect</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>to</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewComponent</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewComponent</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stretch</sender>
   <signal>clicked(bool)</signal>
   <receiver>sizeY</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stretch</sender>
   <signal>clicked(bool)</signal>
   <receiver>sizeZ</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
